app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<section id='recipes' class="flex flex-col justify-center bg-gray-100">
+  <div class="flex flex-col items-center">
```

Comment:
- [ ] Good job so far but I have an issue with your design. I noticed that your recipe list doesn't follow the project wireframe. Please fix it in order to make your app design follows the wireframe 👍 

#### Your design
![recipe](https://github.com/Solobarine/recipes/assets/6615394/e4e6b238-380b-4e1c-961e-2345435cfb7c)

#### The wireframe
![recipe_index](https://github.com/Solobarine/recipes/assets/6615394/bdaceb64-e474-44ef-8eee-8bc8ad3b3d51)